server/statistics/hot_regions_stat.go

```diff
@@ -44,3 +46,12 @@ type HotPeerStatShow struct {
 	AntiCount      int       `json:"anti_count"`
 	LastUpdateTime time.Time `json:"last_update_time"`
 }
+
+// UpdateRegionInfo updates the region information, such as `IsLearner` and `LastUpdateTime`.
+func (h *HotPeerStatShow) UpdateRegionInfo(region *core.RegionInfo) {
```

Comment:
Actually, it updates the hot peer info according to region information?

Reply:
yes, do you have a good idea about its name?

Reply:
Maybe `UpdateHotPeerStat`?